src/models/layerMetadata/layerMetadata.ts

```diff
@@ -37,6 +37,32 @@ export interface IPropPYCSWMapping extends IPYCSWMapping {
 }
 
 export class LayerMetadata implements ILayerMetadata, IMetadataCommonModel {
+  //#region CORE: id
+  @pycsw({
+    profile: 'mc_raster',
+    xmlElement: 'mc:id',
+    queryableField: 'mc:id',
+    pycswField: 'pycsw:Identifier',
+  })
+  @catalogDB({
+    column: {
+      name: 'identifier',
+      type: 'text',
+      nullable: false,
+      primary: true,
+    },
+  })
+  @tsTypes({
+    mappingType: TsTypes.STRING,
+  })
+  @graphql()
+  @fieldConfig({
+    category: FieldCategory.MAIN,
+    isAutoGenerated: true,
```

Comment:
it's not auto generated

Ask @alebinson if some property should be added so it won't be displayed